CMakeLists.txt

```diff
@@ -53,12 +53,17 @@ else()
 endif()
 
 # HDF5 (required)
+set(MGMOL_USE_HDF5P TRUE CACHE BOOL "Use HDF5 parallel capability")
```

Comment:
@jeanlucf22 do we want this to be always true by default?

Reply:
that is making the option being what we have now, by default

Reply:
Oh ok. What about the other comment below? Should changes be applied to HDFrestart.h also?

Reply:
yes, working on it